### DISCUSSION TITLE
Network error

### DISCUSSION THREAD

**Discussion by vitaliy-kovtun-zocdoc** [CMSgov/price-transparency-guide#752] [General]
Hello, when API has changed, I can't download files from new urls, like https://transparency-in-coverage.uhc.com/api/v1/uhc/blobs/download?fd=2024-06-01&fn=2024-06-01_UMR--Inc-_Third-Party-Administrator_WEST-BEND-SCHOOL-DISTRICT_NY-STATEWIDE-NON-EVALUATED-GAP_-A_NXBW_in-network-rates.json.gz

After some time of downloading I get an error: "Connection broken" from python and "Failed - Network error" if I'm trying to download via google chrome browser.

Do you have the same error? In fact, I can't download almost all UHC in-network files from similar URL's

**Comment by shaselton-usds** [comment]
@vitaliy-kovtun-zocdoc Moving to a discussion.

**Comment by jamesonnetworks-clearcost** [comment]
We are seeing this as well. I see it both programmatically trying to download the files and attempting to download the files through a web browser. 

```error: The response ended prematurely. (ResponseEnded),    at System.Net.Http.HttpConnection.FillAsync(Boolean async)```

and 

![unnamed](https://github.com/user-attachments/assets/56ab2e8c-bcf4-417b-b7f8-9cf9d7c90f8f)

**Comment by jamesonnetworks-clearcost** [comment]
@shaselton-usds Are there any steps we should take here to inform UHC of the issues we are seeing? They appear to be systematic for several different files. We find about 7 out of 40 files seem to fail every month.

**Comment by jamesonnetworks-clearcost** [comment]
I believe these issues will be resolved with files on and after Nov 2024
